CONTRIBUTING.md

```diff
@@ -78,11 +78,135 @@ strategy to maintain the project. **PLEASE FOLLOW THIS FORMAT**
   a maintainer asks you to reduce your PR size, please do so.
 - `release/vX.X.X` Used to release the sprints work into production.
 
+## Commit Messages
+
+### Message Structure
+
+A commit message consists of three distinct parts separated by a blank line: the title, an **optional** body and an **optional** footer. The layout looks like this:
+
+`type(scope?): subject`
+
+`Body`
+
+`Footer`
```

Comment:
Let's not use body and footer. We'll try keep it as just the subject :). Bit cleaner